core/tests/test_utils.py

```diff
@@ -415,6 +415,8 @@ def get_json(self, url, params=None, expect_errors=False,
         json_response = self.testapp.get(
             url, params, expect_errors=expect_errors,
             status=expected_status_int)
+
+        self.assertEqual(json_response.status_int, expected_status_int)
```

Comment:
Should we do this for post_json, put_json, etc. as well?

Can we also add a comment above this line explaining why it is needed? (Talk about webtest not doing the right thing in the error case etc.)

Thanks!

Reply:
(And please add a link to the relevant line of code in webtest in a stable versioned copy of the code (i.e. not master). Probably best to focus on the version we actually use.)

Reply:
Oh ok. Will do so too! :)

Reply:
Done.